packages/rx_bloc_cli/lib/src/processors/ios/podfile_processor.dart

```diff
@@ -36,14 +37,19 @@ class PodfileProcessor extends StringProcessor {
     final globalPlatformCommand = 'platform :ios';
     var replacement =
         buffer.toString().replaceAll('# platform :ios', globalPlatformCommand);
+    var minSupportedIOSVersion = _minSupportedIOSVersion;
+    if (args.qrScannerEnabled) {
+      /// The minimum supported iOS version for MLKit is 15.5
+      minSupportedIOSVersion = '15.5';
```

Comment:
We don't need to add such condition. Please set the minimum iOS version to 16.

Reply:
Done